src/ir/enum_ty.rs

```diff
@@ -49,6 +54,11 @@ impl Enum {
         &self.variants
     }
 
+    /// The special attributes of enumeration
+    pub fn attributes(&self) -> &Vec<Attribute> {
```

Comment:
The alignment solution has submitted #468 base on the LLVM API, if that PR could be merged, this one could be simplify, only for document improvement.